framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -40,33 +40,7 @@
 import com.sun.tools.javac.tree.JCTree.JCMemberReference;
 import com.sun.tools.javac.tree.JCTree.JCMemberReference.ReferenceKind;
 import com.sun.tools.javac.tree.TreeInfo;
-import java.lang.annotation.Annotation;
```

Comment:
Most of the changes in this class are code reformatting. The only significant change is removing 674-678